src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/PcsVmrForwardFlower.cs

```diff
@@ -104,4 +105,7 @@ public async Task<bool> FlowForwardAsync(
             discardPatches: true,
             cancellationToken);
     }
+
+    // During forward flow, we're targeting a specific remtoe VMR branch, so we should make sure our local branch is reset to it
+    protected override bool ShouldResetVmr { get; } = true;
```

Comment:
This does not override the parent correctly. It just sets the value to it.
It should be this:
```suggestion
    // During forward flow, we're targeting a specific remote VMR branch, so we should make sure our local branch is reset to it
    protected override bool ShouldResetVmr => true;
```